src/app/workflow/launch-third-party/launch-third-party.component.ts

```diff
@@ -194,6 +215,14 @@ export class LaunchThirdPartyComponent extends Base implements OnChanges, OnInit
     map(([hasContent, hasHttpImports]) => this.sevenBridgesTooltip(hasContent, hasHttpImports, 'Cavatica'))
   );
 
+  colabTooltip$: Observable<string> = combineLatest([this.hasContent$]).pipe(
+    map(([hasContent]) => (hasContent ? 'Run this notebook in Google Colaboratory' : 'The notebook has no content.'))
+  );
+
+  mybinderTooltip$: Observable<string> = combineLatest([this.hasContent$]).pipe(
+    map(([hasContent]) => (hasContent ? 'Run this notebook in My Binder' : 'The notebook has no content.'))
```

Comment:
`My Binder` or `MyBinder`  or `mybinder`?
The docs I can find are annoyingly coy, usually referencing by domain name (mybinder.org)

https://jupytearth.org/jupyter-resources/ecosystem/binder.html or
https://github.com/search?q=repo%3Ajupyterhub%2Fmybinder.org-deploy%20mybinder&type=code